otherlibs/build-info/test/run.t

```diff
@@ -142,3 +142,38 @@ Test substitution when promoting
   lib a: 1.0+a
   lib b: 1.0+b
   lib dune-build-info: XXX
+
+Version is picked from dune-project if available
+------------------------------------------------
+
+  $ echo '(version project-version)' >> c/dune-project
+  $ dune build
+  $ dune install --prefix _install
```

Comment:
We should dump the output to `/dev/null` BTW. It doesn't add much to the test.

Reply:
good idea, done